component/portal/src/main/java/org/exoplatform/portal/rest/GroupRestResourcesV1.java

```diff
@@ -408,6 +408,68 @@ public Response createMembership(MembershipImpl membership) throws Exception {
     return Response.noContent().build();
   }
 
+  @POST
+  @Path("multiMemberships")
```

Comment:
IMO it is better to user this path:
```suggestion
  @Path("memberships/multiple")
```

Reply:
Reference: https://community.exoplatform.com/portal/g/:spaces:engineering/engineering/wiki/REST_CRUD_APIs